src/test/java/model/PlayerTest.java

```diff
@@ -1,29 +1,34 @@
 package model;
 
+import exception.InvalidMoneyAmountException;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class PlayerTest {
+    Player aPlayer;
+
     @BeforeEach
     void setUp() {
-
+        aPlayer = new Player();
+        aPlayer.addCardToHand(new Card(Rank.KING, Suit.DIAMOND));
```

Comment:
addCardToHand doesn't exists, as well as Card, RANK, or Suit. This would never compile